nextcord/types/guild.py

```diff
@@ -34,6 +34,8 @@ class UnavailableGuild(TypedDict):
 VerificationLevel = Literal[0, 1, 2, 3, 4]
 NSFWLevel = Literal[0, 1, 2, 3]
 PremiumTier = Literal[0, 1, 2, 3]
+PromptOptionTypes = Literal[0, 1]
```

Comment:
Would it make sense for this to be singular instead? Weird how they decided to document it as a plural but others are `*Type`.